.github/workflows/netlify.yaml

```diff
@@ -73,7 +53,7 @@ jobs:
               env:
                   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
               with:
-                  pull-request-number: ${{ steps.readctx.outputs.prnumber }}
+                  pull-request-number: ${{github.event.workflow_run.pull_requests[0].number}}
```

Comment:
pretty sure this didn't work before, so yay if they've fixed it!

Reply:
Yeah, I thought it was fairly unlikely I'd missed it before, but couldn't find any concrete point it was introduced.